backend/controllers/auth.controller.js

```
@@ -3,6 +3,7 @@ import User from "../models/user.model.js";
 import bcrypt from "bcryptjs";
 import validator from "validator"; // ✅ Importing validator
 import transporter from "../services/nodemailer.service.js";
+import crypto from "crypto";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)